sdk/communication/communication-call-automation/CHANGELOG.md

```diff
@@ -1,18 +1,27 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
+## 1.3.0-beta.1 (Unreleased)
 
 ### Features Added
-- EventProcessor is a new way to easily handle mid-connection events. Take a look at readme file for usage.
-- StartRecording now accepts PauseOnStart.
-- Introduced `TeamsComplianceRecordingStateChanged` and `TeamsRecordingStateChanged` events.
```

Comment:
What happened with these features?

Reply:
these were not released to any client, so they have changed to as per these PR's https://github.com/Azure/azure-sdk-for-js/pull/29710

Reply:
@joheredi by any chance do you remember which SDK has the link to the PR for the feature? I dont see atleast on the communication sdk's changelog i dont see them included any links.